trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/compute_broadband_noise.py

```diff
@@ -5,119 +5,351 @@
 
 # ----------------------------------------------------------------------
 #  Imports
-# ----------------------------------------------------------------------
-import SUAVE
-from SUAVE.Core import Units , Data 
+# ---------------------------------------------------------------------- 
+from SUAVE.Core import Units , Data
 import numpy as np 
- 
-from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.dbA_noise   import A_weighting  
-from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools             import SPL_harmonic_to_third_octave
 
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.dbA_noise                     import A_weighting
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.SPL_harmonic_to_third_octave  import SPL_harmonic_to_third_octave  
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.decibel_arithmetic            import SPL_arithmetic 
+from SUAVE.Methods.Noise.Fidelity_One.Noise_Tools.compute_source_coordinates    import vectorize
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.compute_naca_4series import compute_naca_4series  
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry \
+     import import_airfoil_geometry
+from SUAVE.Methods.Aerodynamics.Airfoil_Panel_Method.airfoil_analysis           import airfoil_analysis
+from scipy.special import fresnel
+ 
 # ----------------------------------------------------------------------
 # Frequency Domain Broadband Noise Computation
 # ----------------------------------------------------------------------
-## @ingroupMethods-Noise-Fidelity_One-Propeller
-def compute_broadband_noise(freestream,angle_of_attack,position_vector,
+## @ingroupMethods-Noise-Fidelity_One-Propeller   
```

Comment:
move to top of file

Reply:
All codes have this at the top and above the function